### PR TITLE
Add .codacy.yml for python2 config

### PR DESCRIPTION
**Pull request by phil0s0pher**

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -16,65 +22,93 @@
...
-    "voter": "ryan.rabello"
-}, {
-    "id": 3,
-    "updated_at": datetime.now(),
-    "question_id": 3,
-    "voter": "ryan.rabello"
-}]
+
+def gen_askanythings(number=5):
+    """Generate askanythings
```
Comment (N-Perry) [inline tests/utils.py]: ChArLeS!!!!! YoU HaVe A TyPo HeRe!!!!! WhAt Is "AsKaNtHiNgS??!!?!?!

**Comment by phil0s0pher** [comment]
Change did not fix issues with codacy.

File: tests/utils.py [hunk 2]
```diff
@@ -16,65 +22,93 @@
...
+    for i in xrange(number):
+        yield {
+            "id": i,
+            "updated_at": datetime.now(),
+            "question_id": 1,
+            "voter": get_first_name() + '.' + get_last_name()
+        }
+
+
+def edit(generator, changes):
```
Comment (ethanbeaver) [inline tests/utils.py]: Perhaps a more descriptive name, like edit_generator or modify_custom_generator would make the code more readable.

File: tests/utils.py [hunk 3]
```diff
@@ -16,65 +22,93 @@
...
+    for i, record in enumerate(generator):
+        if i in changes.iterkeys():
+            record.update(changes[i])
+
+        yield record
 
 
 @contextmanager
-def askanything(conn, askanythings=ASKANYTHINGS_DATA):
+def askanything(conn, askanythings=None):
```
Comment (ethanbeaver) [inline tests/utils.py]: Exception handling should probably occur in this file.

File: tests/utils.py [hunk 4]
```diff
@@ -16,65 +22,93 @@
...
+        askanythings = list(gen_askanythings())
+
     conn.execute(ASKANYTHING_TABLE.insert(), askanythings)
     yield askanythings
     conn.execute(ASKANYTHING_TABLE.delete())
 
 
 @contextmanager
-def askanthingvote(conn, askanythingvotes=ASKANYTHINGVOTES_DATA):
+def askanthingvote(conn, askanythingvotes=None):
```
Comment (ethanbeaver) [inline tests/utils.py]: Exception handling should probably occur in this file.

**Pull request closed without merging**